src/renderer/containers/Login/LoginContainer.tsx

```diff
@@ -71,23 +72,37 @@ IOwnProps & IInjectedProps & Pick<IRootState, "register">, IState> {
             </Background>
         );
     }
-    
+    private handleNotification = (): boolean => {
+        return this.state.notification!==null ? true : false ;
+    };
+
     private handleChange = (e: React.FormEvent<HTMLInputElement>): void => {
         this.setState({input: e.currentTarget.value});
     };
     
     private handleSubmit = async (): Promise<void> => {
         const account = await database.account.get("account");
```

Comment:
account should be constant. Change it in register action as well

Reply:
I'm not sure what do you mean by this, could you elaborate please.

Reply:
```
await database.account.get(DEFAULT_ACCOUNT);
// and change database save in register action to use that constant as well instead of "account"
```